orchestrator/relayer/src/main_loop.rs

```diff
@@ -99,7 +100,15 @@ pub async fn relayer_main_loop(
 
                 Ok(())
             },
-            sleep(TIMEOUT)
+            async {
+                // a bit of logic that tires to keep things running every relayer_loop_speed seconds exactly
```

Comment:
typo, should be "tries" not "tires"

Reply:
Just removed that comment, it's redundant